google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobException.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2017, Google LLC All rights reserved.
```

Comment:
Nit: Per https://opensource.google.com/docs/releasing/preparing/#include-license-file-and-source-code-headers the top line should be `Copyright 2017 Google LLC` (no `All rights reserved.`)

Reply:
Done